lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -36,9 +36,45 @@ function collectImageElementInfo() {
       clientHeight: element.clientHeight,
       naturalWidth: element.naturalWidth,
       naturalHeight: element.naturalHeight,
+      isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
     };
   });
+
+  // Only match basic background-image: url("http://host/image.jpeg") declarations
+  const ALLOWED_IMAGE = /^\s*url\("([^"]+)"\)\s*$/;
```

Comment:
in getComputedStyle Chrome normalizes the URL to full URL in quotes :) in fact the spacing is probably totally useless too since I bet those are eliminated as well (just not as thoroughly tested by me)